src/main/webapp/web.xml

```diff
@@ -11,4 +11,8 @@
         <servlet-name>index</servlet-name>
         <url-pattern>/index</url-pattern>
     </servlet-mapping>
+    <servlet-mapping>
+        <servlet-name>index</servlet-name>
+        <url-pattern>/</url-pattern>
+    </servlet-mapping>
```

Comment:
If you use @WebServlet annotation, it's better to use it everywhere(better to add it to IndexController too)